analysers/analyser_merge_parking_FR_BNLS.py

```diff
@@ -0,0 +1,111 @@
+## Copyrights Vincent Bergeot (thanks Frédéric Rodrigo) 2020             ##
+##                                                                       ##
+## This program is free software: you can redistribute it and/or modify  ##
+## it under the terms of the GNU General Public License as published by  ##
+## the Free Software Foundation, either version 3 of the License, or     ##
+## (at your option) any later version.                                   ##
+##                                                                       ##
+## This program is distributed in the hope that it will be useful,       ##
+## but WITHOUT ANY WARRANTY; without even the implied warranty of        ##
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         ##
+## GNU General Public License for more details.                          ##
+##                                                                       ##
+## You should have received a copy of the GNU General Public License     ##
+## along with this program.  If not, see <http://www.gnu.org/licenses/>. ##
+##                                                                       ##
+###########################################################################
+
+from .Analyser_Merge import Analyser_Merge, Source, CSV, Load, Mapping, Select, Generate
+
+
+class Analyser_Merge_Parking_FR_BNLS(Analyser_Merge):
+    def __init__(self, config, logger = None):
+        Analyser_Merge.__init__(self, config, logger)
+            doc1 = dict(
+                title = T_('Opendata'),
+                detail = T_(
+''''This is a data from an opendatabase, without any verification'''),
+                fix = T_(
+'''If you're sure that it's a new data for OpenStreetMap, then you can add it with your favorite editor (picture of Id, josm link ?)'''),
+                trap = T_(
+'''It's not because it's open that it's right,see it's because it's opendata that OSM contributors needs to check '''),
+            doc3 = dict(
+                title = T_('Integration suggestion'),
+                detail = T_(
+''''This is a integration suggestion, mixing opendatabase and OpenStreetMap.'''),
+                fix = T_(
+'''If you're sure that it's a good integration then you can add it with your favorite editor (picture of Id, josm link ?)'''),
+                trap = T_(
+'''It's not because there's a integration suggestion that it's right, you're OSM contributors and not a machine !'''),
+            doc4 = dict(
+                title = T_('Update'),
+                detail = T_(
+''''This is an update suggestion because there's the same ref in opendatabase and OSM.'''),
+                fix = T_(
+'''If you're sure that it's a good update then you can add it with your favorite editor (picture of Id, josm link ?)'''),
+                trap = T_(
+'''It's not because there's a update suggestion that it's right, you're OSM contributors and not a machine !''')
```

Comment:
```suggestion
''''This is a data from an opendatabase, without any verification'''),
                fix = T_(
'''If you are sure that it's a new data for OpenStreetMap, then you can add it with your favourite editor (picture of Id, josm link ?).'''),
                trap = T_(
'''It is not because it is open that it is good data, see it is because it is opendata that OSM contributors needs to check.'''),
            doc3 = dict(
                title = T_('Integration suggestion'),
                detail = T_(
''''This is a integration suggestion, mixing opendatabase and OpenStreetMap.'''),
                fix = T_(
'''If you are sure that it is a good integration then you can add it with your favourite editor (picture of Id, josm link ?).'''),
                trap = T_(
'''It is not because there is an integration suggestion that it is right, you are an OSM contributor and not an integration machine.'''),
            doc4 = dict(
                title = T_('Update'),
                detail = T_(
''''This is an update suggestion because there is the same ref in opendatabase and OSM.'''),
                fix = T_(
'''If you are sure that it is a good update then you can add the tag, or part of, it with your favourite editor (picture of Id, josm link ?)'''),
                trap = T_(
'''It is not because there is a update suggestion that it is right, you are an OSM contributor and not an integration machine.''')
```